lib/features/down_by_maintenance/presentation/down_by_maintenance_wm.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:elementary/elementary.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_template/features/down_by_maintenance/data/services/mock_check_server_status_service.dart';
+import 'package:flutter_template/features/down_by_maintenance/domain/entities/check_result.dart';
+import 'package:flutter_template/features/down_by_maintenance/domain/services/check_server_status_service.dart';
+import 'package:flutter_template/features/down_by_maintenance/presentation/down_by_maintenance_widget.dart';
+
+/// Factory that creates DownByMaintenanceWM.
+DownByMaintenanceWM createDownByMaintenanceWM(BuildContext context) {
+  return DownByMaintenanceWM(MockCheckServerStatusService());
+}
+
+/// WidgetModel for DownByMaintenanceWidget.
+class DownByMaintenanceWM extends WidgetModel<DownByMaintenanceWidget, _StubModel> implements IDownByMaintenanceWM {
+  final ICheckServerStatusService _checkServerStatusService;
+
+  @override
+  ValueListenable<ServerCheckResult> get currentResult => _checkServerStatusService.serverStatus;
+
+  /// @nodoc
+  DownByMaintenanceWM(this._checkServerStatusService) : super(_StubModel());
+
+  @override
+  void initWidgetModel() {
+    _checkServerStatusService.configurate();
```

Comment:
It is business logic and should be in the `ElementaryModel`.

Reply:
I think we should call `initImmediateCheck` as soon as possible after application start.

Reply:
\+